addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -168,16 +168,16 @@
                         </div>
                         <div t-att-id="category.id === 'snippet_custom' ? 'snippet_custom_body' : ''" class="o_panel_body" t-on-pointerup="_onMouseUp">
                             <t t-foreach="category_value" t-as="snippet" t-key="snippet.id">
+                                <t t-set="disabledTooltip">This block cannot be dropped anywhere on this page.</t>
                                 <div t-if="snippet.visible"
                                     class="oe_snippet"
                                     t-att-class="{ 'o_disabled': snippet.disabled, 'o_snippet_install': snippet.installable, 'o_we_draggable': !snippet.renaming and !snippet.installable and !snippet.disabled }"
                                     t-att-name="snippet.displayName"
                                     t-att-data-oe-snippet-id="snippet.id"
                                     t-att-data-module-id="snippet.moduleId"
-                                    t-on-click="_onSnippetClick"
-                                    data-tooltip="This block cannot be dropped anywhere on this page.">
```

Comment:
We generally cannot remove/modify attributes from XML templates in stable, we can only add new ones
(just telling so you know) Here it is okay because it is a static file, but it is always "au cas par cas" so always make sure that you can :+1: 

Depending on how you will adapt based on what I said in my previous comment, I guess you could fix that by adding a `t-att-data-tooltip` right after this line (like you did on the image), I think its value will overwrite `data-tooltip` but I am not sure :wink:

Reply:
Adding the `t-att-data-tooltip` just after `data-tooltip` overwrite the behaviour. 
```
data-tooltip="This block cannot be dropped anywhere on this page."
t-att-data-tooltip="(snippet.disabled || snippet.installable) ? disabledTooltip : false"
```